root/static/scripts/common/components/TagEditor.js

```diff
@@ -380,17 +387,21 @@ class SidebarTagEditor extends TagEditor {
   render() {
     return (
       <div>
-        <ul className="tag-list">
-          {this.createTagRows()}
-          {this.props.more
-            // createTagRows uses tags as keys; \0 is simply used here to
-            // guarantee that there isn't any conflict.
-            ? <li key="\0:more">
-                <a href={getTagsPath(this.props.entity)}>{l('more...')}</a>
-              </li>
-            : null}
-        </ul>
-        {!this.state.tags.length && <p>{lp('(none)', 'tag')}</p>}
+        {this.state.tags.length ? (
+          <ul className="tag-list">
+            {this.createTagRows()}
+            {this.props.more
+              // createTagRows uses tags as keys; \0 is simply used here to
+              // guarantee that there isn't any conflict.
+              ? <li key="\0:more">
+                  <a href={getTagsPath(this.props.entity)}>{l('more...')}</a>
+                </li>
+              : null}
+          </ul>
+        ) : (
+          <p>{lp('(none)', 'tag')}</p>
```

Comment:
There's no vertical space between Tags and (none), unlike in production. There probably should be.